python/py-sybil/Portfile

```diff
@@ -17,31 +17,26 @@ long_description    This library provides a way to test examples in your \
                     your normal test run. Integration is provided for the \
                     three main Python test runners.
 
-homepage            https://pypi.python.org/pypi/${python.rootname}
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}/
-distname            ${python.rootname}-${version}
+homepage            https://github.com/simplistix/sybil
 
-checksums           rmd160  5fa7b4e6b9fba4dd807bfbf4850b804be8caa016 \
-                    sha256  597d71e246690b9223c132f0ed7dcac470dcbe9ad022004a801e108a00dc3524 \
-                    size    29316
+checksums           rmd160  c4a72e2d29f973ca006d36812bc81c22d522a326 \
+                    sha256  7692ed66e793e5e79ae6a70cf2cf861917ed48eaff0d8adf825e64d85820f251 \
+                    size    38011
 
-python.versions     27 35 36 37 38
+python.versions     27 37 38 39
```

Comment:
this new version doesn't support PY27 anymore; please account for this (and check this for the other PRs you submitted as well)

Reply:
Thank you. Done.